communication/slack-guidelines.md

```diff
@@ -83,12 +84,12 @@ development to kubernetes,
 ### Escalating and/or Reporting a Problem
 
 The Kubernetes Slack has an integrated tool for reporting issues. It may be
-accessed by clicking on "More actions", the "`...`" to the right of a message,
+accessed by clicking on "More actions", the "**...**" to the right of a message,
```

Comment:
Not necessary for this PR, but would be nice to have a screenshot here :stuck_out_tongue: